### PR TITLE
Update README Install with khd instructions

### PR DESCRIPTION
**Pull request by jgillman**
Makes khd "dependency" more clear for users who are looking for an "all in one" solution.

Hope this clears up #512.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,26 @@ Manage *Kwm* using brew services
...
+
+Add the example [`kwmrc`](https://github.com/koekeishiya/kwm/blob/master/examples/kwmrc) file to `~/.kwm/kwmc` to get started.
+
+Manage Kwm using brew services
+
+    brew services start kwm
+
+## Caveats
+
+Kwm does not support keyboard shortcuts out of the box. If you intend on using keyboard shortcuts to control Kwm you can install [Khd](https://github.com/koekeishiya/khd). There is an [example `.khdrc` file in the Kwm source](https://github.com/koekeishiya/kwm/blob/master/examples/khdrm) you can use to get started by adding it to `~/.khdrc`.
```
Comment (dominiklohmann) [inline README.md]: Typo: the second link here should be https://github.com/koekeishiya/kwm/blob/master/examples/khdrc instead

**Comment by jgillman** [comment]
@dominiklohmann Fixed that typo and removed the previous `# Install` block which I missed the first time!

**Comment by koekeishiya** [comment]
> As of version 3 Kwm does not support keyboard shortcuts

This should be version 4.0.1 IIRC.

**Comment by koekeishiya** [comment]
> file to `~/.kwm/kwmc` 

This should be `~/.kwm/kwmrc`


Can you also change the `Install` header to be `## Install`

**Comment by koekeishiya** [comment]
Looks fine now.

Can you rebase and add a proper commit message?

Thanks.

**Comment by jgillman** [comment]
@koekeishiya Done! I've squashed my commits and cleaned up the commit message. Thanks for the feedbacks.

**Pull request merged**